### PR TITLE
Add function name to debug output

### PR DESCRIPTION
**Pull request by 3djc**

### CODE REVIEW

File: radio/src/lua/interface.cpp [hunk 1]
```diff
@@ -981,7 +981,7 @@ class LuaWidgetFactory: public WidgetFactory
       }
 
       if (lua_pcall(L, 2, 1, 0) != 0) {
-        TRACE("Error in widget %s: %s", getName(), lua_tostring(L, -1));
+        TRACE("Error in create() of widget %s: %s", getName(), lua_tostring(L, -1));
```
Comment (bsongis) [inline radio/src/lua/interface.cpp]: @kilrah @projectkk2glider is it good english? we both are french :)

Comment (projectkk2glider) [inline radio/src/lua/interface.cpp]: I am no expert :blush:, but it sounds weird. My suggestion: `Error in widget %s create() funciton: %s`

Comment (bsongis) [inline radio/src/lua/interface.cpp]: I thought of something like this, yes. Without the typo ;)

Comment (projectkk2glider) [inline radio/src/lua/interface.cpp]: :stuck_out_tongue:

File: radio/src/lua/interface.cpp [hunk 2]
```diff
@@ -981,7 +981,7 @@ class LuaWidgetFactory: public WidgetFactory
       }
 
       if (lua_pcall(L, 2, 1, 0) != 0) {
-        TRACE("Error in widget %s: %s", getName(), lua_tostring(L, -1));
+        TRACE("Error in widget %s create() function : %s", getName(), lua_tostring(L, -1));
```
Comment (bsongis) [inline radio/src/lua/interface.cpp]: no space before : in english (not the same than french)

**Comment by bsongis** [comment]
Yes it should be OK. Just the typos

**Comment by bsongis** [comment]
Thanks!

**Pull request merged**